### ISSUE TITLE
JetBrains.ExternalAnnotations missing on nuget

### ISSUE THREAD

**Issue by bkoelman** [JetBrains/ExternalAnnotations#159]
It looks like JetBrains.ExternalAnnotations was unlisted from [nuget.org](https://www.nuget.org/packages/JetBrains.ExternalAnnotations/) recently. My [analyzer](https://github.com/bkoelman/ResharperCodeContractNullability/#running-on-your-build-server) has a dependency on it. Please keep distributing future versions on nuget.org for easy install.

**Comment by derigel23** [comment]
Hi @bkoelman !

Why do your package depends on ExternalAnnotations package? It's internal package (though builded from public sources https://github.com/JetBrains/ExternalAnnotations). It's not a plugin in a classic ReSharper way. It's updated by special mechanism from ReSharper Gallery, not nuget.org.
It's also not a classic nuget package to install in to a project.

It's not a big deal to unlist it - just to understand  your usages of it.

**Comment by bkoelman** [comment]
I'm the author of [ResharperCodeContractNullability](https://github.com/bkoelman/ResharperCodeContractNullability), which is a roslyn-based code analyzer that reports all possible locations in a C# codebase where Resharpers' nullability attributes (`NotNull`/`CanBeNull`/`ItemNotNull`/`ItemCanBeNull`) can be added. Once a user starts adding those attributes to their codebase, the Resharper nullability engine becomes even more effective, leading to less `NullReferenceExceptions` at runtime.

Because nullability attributes on base types and interfaces are inherited, the analyzer needs to know about external annotations. Primarily those of the BCL, but potentially also other popular libraries. So it tries to load them from the Resharper installation folder.

This all works well, except in two scenarios:
1. Resharper may not be installed on the build server. Adding a package reference to `JetBrains.ExternalAnnotations` plus a post-build event that copies the external annotation files to an expected location makes the analyzer work there.
2. In open-source projects, not all contributors may have Resharper installed. Adding the package reference ensures the external annotations can be found in the NuGet cache folder (`%USERPROFILE%\.nuget`), where the analyzer can read them from. Note this approach also solves the build server issue.

In both cases above, the build breaks when building with warnings-as-errors.

So to summarize, the package reference is needed so that the analyzer can read the external annotations to do its work. Hope that makes sense. Feel free to ask for any clarifications if needed.

**Comment by derigel23** [comment]
Package is listed.

**Comment by bkoelman** [comment]
Thanks!
